test/simulator/timestamp/src/connection_simulator.cpp

```diff
@@ -66,21 +67,17 @@ connection_simulator::open_session()
     return (session);
 }
 
-bool
+void
 connection_simulator::close_session(session_simulator *session)
 {
-    std::vector<session_simulator *>::iterator position =
-      std::find(_session_list.begin(), _session_list.end(), session);
+    auto position = std::find(_session_list.begin(), _session_list.end(), session);
 
-    /* If session doesn't exist in the session_list return false. */
-    if (position == _session_list.end()) {
-        std::cerr << "Error: session is not present in the session list" << std::endl;
-        return (false);
-    }
+    /* The session to be closed should be present in the session list. */
+    assert(position != _session_list.end());
 
     _session_list.erase(position);
     delete session;
-    return (true);
+    session = NULL;
```

Comment:
`nullptr` should be used here

Reply:
Thanks, updated in [d634513](https://github.com/wiredtiger/wiredtiger/pull/8167/commits/d6345130894b574d404ccf4daf81ffdffd54a2ec)